src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Azure.Base.Attributes;
+
+[assembly:AzureSdkClientLibrary("config")]
```

Comment:
How did you choose the name? We need to have some consistent way to map from library names to this string. I am ok with merging this PR, but please file an issue (high priority) to answer this question.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/5618